packages/ui-react/src/components/Epg/Epg.tsx

```diff
@@ -64,19 +64,23 @@ export default function Epg({ channels, onChannelClick, onProgramClick, channel,
         <Layout
           {...getLayoutProps()}
           renderTimeline={(props) => <EpgTimeline {...props} />}
-          renderChannel={({ channel: epgChannel }) => (
-            <EpgChannelItem
-              key={epgChannel.uuid}
-              channel={epgChannel}
-              channelItemWidth={channelItemWidth}
-              sidebarWidth={sidebarWidth}
-              onClick={(toChannel) => {
-                onChannelClick(toChannel.uuid);
-                onScrollToNow();
-              }}
-              isActive={channel?.id === epgChannel.uuid}
-            />
-          )}
+          renderChannel={({ channel: epgChannel }) => {
+            const channel = channels.find((channel) => channel.id === epgChannel.uuid);
+            return (
+              <EpgChannelItem
+                key={epgChannel.uuid}
+                channel={epgChannel}
+                channelItemWidth={channelItemWidth}
+                sidebarWidth={sidebarWidth}
+                onClick={(toChannel) => {
+                  onChannelClick(toChannel.uuid);
+                  onScrollToNow();
+                }}
+                title={channel?.title || ''}
+                isActive={!!channel}
```

Comment:
This might not be clear from reading the arguments/props, but the Epg channel prop is the currently selected/active channel.

The `isActive` prop should be set to `true` for only the active channel, but currently will be set to `true` for all channels. Maybe we can rename the prop to `activeChannel` or `selectedChannel` (the term active is already used here).

Reply:
While i was laying on my pillow, thinking about this, I realized this too and opened my laptop to have a look at it and saw you noticed this too 37 min ago haha. I will fix this tomorrow ;)